com.ibm.streamsx.json/impl/java/src/com/ibm/streamsx/json/converters/JSONToTupleConverter.java

```diff
@@ -0,0 +1,404 @@
+package com.ibm.streamsx.json.converters;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+import com.ibm.json.java.JSONArray;
+import com.ibm.json.java.JSONObject;
+import com.ibm.streams.operator.Attribute;
+import com.ibm.streams.operator.StreamSchema;
+import com.ibm.streams.operator.Tuple;
+import com.ibm.streams.operator.Type;
+import com.ibm.streams.operator.logging.TraceLevel;
+import com.ibm.streams.operator.meta.CollectionType;
+import com.ibm.streams.operator.meta.TupleType;
+import com.ibm.streams.operator.types.RString;
+import com.ibm.streams.operator.types.Timestamp;
+
+public class JSONToTupleConverter {
+
+	private static Logger l = Logger.getLogger(JSONToTupleConverter.class.getCanonicalName());
+	
+	/**
+	 * Convert a JSON object to a Java object of the specified type. 
+	 * 
+	 * @param name The name of the attribute being converted (used for logging purposes)
+	 * @param type SPL type that the JSON object is being converted to. 
```

Comment:
Should this be MetaType? Or have overloaded versions with the Type one just calling the MetaType one?